geanynumberedbookmarks/src/Makefile.am

```diff
@@ -1,6 +1,4 @@
+include $(top_srcdir)/build/vars.init.mk
 include $(top_srcdir)/build/vars.build.mk
-
-geanyplugins_LTLIBRARIES = geanynumberedbookmarks.la
-
-geanynumberedbookmarks_la_SOURCES = geanynumberedbookmarks.c
-geanynumberedbookmarks_la_LIBADD = $(COMMONLIBS)
+include $(top_srcdir)/build/cppcheck.mk
```

Comment:
nice-ish, but cppcheck checking was opt-in.  Maybe we want it to be enabled for everyone now, but that's probably not for this PR to decide